### PR TITLE
[#11] Add component: flash

### LINKED ISSUE #11: Add component: flash
See https://bitcrowd.github.io/bitstyles/?path=/docs/ui-flashes--flash-brand-1

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #11 

- Adds a flash component
- Drive-by update to the README, correcting the version number of bitstyles that we currently import

TODO:
- [] Add icons

Looks like:
Screenshot 2021-04-13 at 11 04 50

### CODE REVIEW

File: lib/bitstyles_phoenix.ex [hunk 1]
```diff
@@ -33,4 +33,22 @@ defmodule BitstylesPhoenix do
...
+    ui_flash("Changes succesfully saved", variant: "positive")
+  """)
+
+  showcase("Warning", """
+    ui_flash("Login not successful, please try that again", variant: "warning")
+  """)
+
+  showcase("Danger", """
+    ui_flash("An error occured, please contact an admin", variant: "danger")
+  """)
```
Comment (andreasknoepfle) [inline lib/bitstyles_phoenix.ex:53]: We should add one unit test and showcase for the `do` option like this (i think): 

```
ui_flash("An error occured, please contact an admin", variant: "danger") do
  content_tag(:span, "test")
end
```

Comment (planktonic) [reply lib/bitstyles_phoenix.ex:53]: Yeh I never got the to work (would be great for buttons too!). Sadly I can’t get this to work in the showcase nor the doctests 🤔 Probably I’m just getting the nesting/syntax wrong…

Comment (andreasknoepfle) [reply lib/bitstyles_phoenix.ex:59]: Added :)

**Comment by andreasknoepfle** [comment]
Let's add the icons in a seperate PR :)

**Pull request merged**